static/docs/command-reference/get.md

```diff
@@ -1,28 +1,40 @@
 # get
 
-Download or copy file or directory from the
-[remote storage](/doc/command-reference/remote) of any <abbr>DVC project</abbr>
-in a Git repository (e.g. hosted on GitHub) into the current working directory.
+Download a file or directory from any <abbr>DVC project</abbr> or Git repository
+(e.g. hosted on GitHub) into the current working directory.
 
-> Unlike `dvc import`, this command does not track the downloaded data files
-> (does not create a DVC-file).
+> Unlike `dvc import`, this command does not track the downloaded files (does
+> not create a DVC-file).
 
 ## Synopsis
 
 ```usage
 usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
+Download files or directories from DVC repository.
+Documentation: <https://man.dvc.org/get>
+
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url                   URL of Git repository with DVC project to
+                        download from.
+  path                  Path to a file or directory within a
+                        DVC repository.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -q, --quiet           Be quiet.
+  -v, --verbose         Be verbose.
+  -o [OUT], --out [OUT]
+                        Destination path.
+  --rev [REV]           DVC repository git revision.
```

Comment:
Same with all this. We edit for better looking in the cmd ref. And no need to include the options since they are described in detail later in this doc. Let's revert this whole chunk please.